### PR TITLE
Adding featured image credits and drafting the first post

### PR DESCRIPTION
**Pull request by merit-mthompson**
Hey team,

Tagging everyone I can for visibility of our new site and what a PR looks like for a new post. We'll need at least one reviewer approval for PR merge and once a PR is merged the new content will be automatically built and posted to https://merit-network.github.io/. (What's up on the site now is just a placeholder, it isn't this first post.)

I can tell already that we need a way to visually review these posts outside of code and that's something we can work on in the near future. For now, please review the changes in this PR and the attached screenshot of the post.

If there are changes we think we should make to this site beyond the content in this PR please open an issue and we'll get things resolved (just like we do with our other stuff on GitHub).

I'm documenting all of this in Confluence; still finalizing that and not going to share the link publicly.

Thanks!
Michael

Screenshot_2020-10-16 Hello World

### CODE REVIEW

File: content/blog/hello-world/index.md [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
+* [Perl](https://www.perl.org/)
+* [Python](https://www.python.org/)
+* [React](https://reactjs.org/)
+* [Routing Assets Database](https://www.radb.net/)
+* [Zenoss](https://www.zenoss.com/)
+* _...and more!_
+
+# This entire site is open source
+
+As a start, [this entire site is open source](https://github.com/merit-network/merit-network.github.io) and MIT-licensed! It was built using [gatsby-starter-blog](https://github.com/gatsbyjs/gatsby-starter-blog) along with [Bulma](https://www.gatsbyjs.com/) and [Font Awesome](https://fontawesome.com/) and uses GitHub Actions for automatic builds and GitHub Pages for hosting.
```
Comment (rcluk) [inline content/blog/hello-world/index.md:34]:
```suggestion
As a start, [this entire site is open source](https://github.com/merit-network/merit-network.github.io) and MIT-licensed! It was built using [gatsby-starter-blog](https://github.com/gatsbyjs/gatsby-starter-blog) along with [Bulma](https://bulma.io/) and [Font Awesome](https://fontawesome.com/) and uses GitHub Actions for automatic builds and GitHub Pages for hosting.
```

**Review by schelcj — APPROVED** [review]

File: src/templates/blog-post.js [hunk 2]
```diff
@@ -34,16 +34,34 @@ const PostHeader = ({ author, post }) => {
 }
 
 const PostFeaturedImage = ({ post }) => {
-  const featuredImgFluid = post.frontmatter?.featuredImage?.childImageSharp?.fluid
-
-  if (!featuredImgFluid) return (null);
+  const featuredImage = post.frontmatter?.featuredImage?.childImageSharp?.fluid
```
Comment (rcluk) [inline src/templates/blog-post.js:37]: I've never actually seen/noticed the "?" in JS expressions before - handy!
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

**Review by rcluk — APPROVED** [review]

File: src/templates/blog-post.js (hunk 2 above)
Comment (merit-mthompson) [reply src/templates/blog-post.js:37]: Yeah those were something new to me too; don't think I'll ever not use them now :D

File: content/blog/hello-world/index.md (hunk 1 above)
Comment (merit-mthompson) [reply content/blog/hello-world/index.md]: Good catch! Added.

**Review by merit-dsclair — APPROVED** [review]

**Review by merit-dsclair — APPROVED** [review]

**Review by merit-jrussell — APPROVED** [review]

**Pull request merged**